cmake/Modules/FindPytorch.cmake

```
@@ -24,18 +24,18 @@ endif()
 
 execute_process(COMMAND ${PY_EXE} -c "import torch; from torch.utils.cpp_extension import CUDA_HOME; print(True if ((torch.version.cuda is not None) and (CUDA_HOME is not None)) else False)"
                 OUTPUT_VARIABLE Pytorch_CUDA OUTPUT_STRIP_TRAILING_WHITESPACE)
-string(REGEX REPLACE "No CUDA runtime[^\n]*\n?" "" Pytorch_CUDA "${Pytorch_CUDA}")
+string(REGEX REPLACE "(No (CUDA|ROCm) runtime[^\n]*\n)" "" Pytorch_CUDA "${Pytorch_CUDA}")
```

```suggestion
string(REGEX REPLACE "No (CUDA|ROCm) runtime[^\n]*\n" "" Pytorch_CUDA "${Pytorch_CUDA}")
```
These brackets are not needed, I think.

---

Updated